api/messaging/src/index.ts

```diff
@@ -12,18 +12,19 @@ export type {
 } from "./types"
 
 /**
- * Should only be called from CS or Ext Pages
- * Extension Id is required to send a message from a CS in the main world
- * TODO: Add a framework runtime check, using a global variable
+ * Send to Background Service Workers from Content Scripts or Extension pages.
```

Comment:
Modifications to this file do not seem to be related to this commit. What do you think about splitting them into two commits to avoid noise in its history ?